apps/site/components/Common/Select/index.module.css

```
@@ -96,7 +96,9 @@
   }
 
   .text > span > span {
-    @apply truncate;
+    @apply max-w-[250px]
```

```suggestion
    @apply max-w-64
```
Instead of giving px you can use `max-w-64` which corresponds to 16rem(256px)

---

sure

---

done cc @canerakdas